ui/analyse/src/ground.ts

```diff
@@ -121,6 +121,10 @@ export function makeConfig(ctrl: AnalyseCtrl): CgConfig {
             ? 'https://playstrategy.org/assets/piece/xiangqi/' +
               d.pref.pieceSet.filter(ps => ps.gameFamily === 'xiangqi')[0].name +
               '/'
+            : ctrl.data.game.gameFamily === 'breakthroughtroyka'
+            ? 'https://playstrategy.org/assets/piece/chess/' + // @TODO: use https://playstrategy.org/assets/piece/breakthrougtroyka/ instead
```

Comment:
Need to upload some assets (and create static pages to describe rules for both games)

Reply:
I can add those rules pages to prismic, I normally like to add a screen shot of the starting position with our default board and color, so I can do that once its on dev.